### PR TITLE
Clean after tests

### LINKED ISSUE #48: Clean up after acceptance tests
## Problem
When running the acceptance tests, two folders are created: `visual-report` and `visual-test`.
When running the second time, the tests could pass even if the code is broken.

## Solution
Clean up after testing.

### PR DESCRIPTION
**Pull request by adela-istrate-hs**
Closes #48

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs  @valentin-radulescu-hs can you review?

File: tests/acceptance/helpers.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import fs from 'fs-extra';
+
+module.exports = {
+  cleanUp: function(path, done) {
```
Comment (andrei-picus-hs) [inline tests/acceptance/helpers.js]:
```js
export function cleanup() { };
import {cleanup} from 'helpers';
```

File: tests/acceptance/helpers.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    fs.remove(path, function(error) {
+      if (error && error.code !== 'ENOENT') {
+        throw error;
+      }
+      done();
```
Comment (andrei-picus-hs) [inline tests/acceptance/helpers.js]: Let's return a promise instead.

```js
return new Promise((resolve, reject) => {
  fs.remove(path, error => {
    if (error) return reject(error);
    resolve();
  });
});
```

File: tests/acceptance/setup/test.js [hunk 3]
```diff
@@ -21,14 +22,17 @@ describe('Generate a dummy report for testing', function() {
     let browser, mugshot, webdriverioInstance;
 
     before(function(done) {
-      webdriverioInstance = webdriverio.remote(BROWSER_OPTIONS).init()
-        .url(URL)
-        .then(function() {
-          browser = new WebdriverIOAdapter(webdriverioInstance);
-          mugshot = new Mugshot(browser, MUGSHOT_OPTIONS);
+      cleanUp('visual-report', function() {
```
Comment (andrei-picus-hs) [inline tests/acceptance/setup/test.js]: Let's clean up after the tests run.

**Comment by andrei-picus-hs** [comment]
Finished review.

File: tests/acceptance/setup/test.js (hunk 3 above)
Comment (adela-istrate-hs) [inline tests/acceptance/setup/test.js]: This is the test file that creates the report.

I am cleaning up in the test file which tests the reporter:
https://github.com/uberVU/mocha-mugshot-reporter/blob/48-clean-after-tests/tests/acceptance/test.js#L75

Comment (andrei-picus-hs) [inline tests/acceptance/setup/test.js]: I see. Please add a comment about this.

**Comment by valentin-radulescu-hs** [comment]
LGTM

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs  Do you have other suggestions?

File: tests/acceptance/helpers.js [hunk 4]
```diff
@@ -1,12 +1,13 @@
 import fs from 'fs-extra';
 
-module.exports = {
-  cleanUp: function(path, done) {
-    fs.remove(path, function(error) {
+export function cleanUp(path) {
+
```
Comment (andrei-picus-hs) [inline tests/acceptance/helpers.js]: Remove.

File: tests/acceptance/test.js [hunk 5]
```diff
@@ -72,8 +72,10 @@ describe('Mocha Mugshot acceptance tests', function() {
     });
 
   after(function(done) {
-    cleanUp('visual-report', function() {
-      done();
-    });
+    // Clean up after running the acceptance tests
+    cleanUp('visual-report')
+      .then(function() {
+        done();
```
Comment (andrei-picus-hs) [inline tests/acceptance/test.js]: No need for this. Just return the promise and Mocha will handle it.

**Comment by andrei-picus-hs** [comment]
Finished review.

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs can I merge?

**Comment by andrei-picus-hs** [comment]
LGTM

**Pull request merged**